### DISCUSSION TITLE
Separate audio volume for Effect UI and Hitsound

### DISCUSSION THREAD

**Discussion by AlexHgC** [ppy/osu#15862] [Ideas] [answered]
Hey! I just ask if it would be technically possible to separate audio effects and hitsound?

I'm a mania player, and like many others, I don't like playing with hitsound. So I have to turn down the Effects volume to reduce the volume of the hitsound in-game. However, this also lowers the menu sound, so I have no way to play without hitsound and with sounds once in the menu.

I sincerely think this feature would be useful for a lot of players and not just for mania players.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/14529
